pkg/controller/nodelifecycle/scheduler/taint_manager_test.go

```diff
@@ -306,15 +303,14 @@ func TestUpdatePod(t *testing.T) {
 			podIndexer.Add(item.prevPod)
 			controller.PodUpdated(nil, item.prevPod)
 
-			fakeClientset.ClearActions()
```

Comment:
this line is not needed anymore?

Reply:
I don't think it was needed on master - deleted and the test continue to pass on master and on the branch.
The first `PodUpdate` does not trigger deletion.